code/ZAS/Radon.dm

```diff
@@ -6,5 +6,7 @@
 	if(!src.loc)
 		return
 	
-	var/molesofradon=src.loc.return_air().molar_density(GAS_RADON)*CELL_VOLUME
+	
+	var/datum/gas_mixture/environment=src.loc.return_air()
+	var/molesofradon=(environment?.molar_density(GAS_RADON) || 0)  *CELL_VOLUME
 	src.apply_radiation(molesofradon*1.5, RAD_EXTERNAL)
```

Comment:
If enviornment doesn't exist, then this will also be a null * 1.5 statement... cursed.

Reply:
Nevermind I missed ``|| 0`` which if the first part is null, will use the last false value as the definition. Thus, 0. We're good.